vsts_ci/azure-pipelines.yml

```diff
@@ -27,16 +27,27 @@ stages:
     steps:
     - template: raspi/continuous-raspi.yml
 
-  - job: PublishFile
+  - job: GithubRelease
+    dependsOn:
+      - LinuxUnitTests
+      - RaspberryPiUnitTests
+    condition: and(succeeded('LinuxUnitTests'), succeeded('RaspberryPiUnitTests')) 
     steps:
-    - task: CopyFiles@2
+    - script: |  
+        mkdir dest
+        cp src/azure-iot-edge-installer.sh dest/ 
+      displayName: Create dest/ directory
+    - task: GitHubRelease@1
+      displayName: 'GitHub release (create)'
       inputs:
-        SourceFolder: '$(BUILD.REPOSITORY.LOCALPATH)/release'
-        TargetFolder: '$(Build.ArtifactStagingDirectory)'
-      displayName: "Copy files to build artifact staging directory"
-    
-    - task: PublishBuildArtifacts@1
+        gitHubConnection: 'github.com_cindydeng1998'
```

Comment:
what if 'github.com_cindydeng1998' decides to move on to a different project?

Reply:
We don't have a team Github account for now, and we can always change the service connection if 'github.com_cindydeng1998' decides to move on to a different project :)